### ISSUE TITLE
Error when uploading file in media

### ISSUE THREAD

**Issue by haritsandrean** [knadh/listmonk#620]
**Version:**
 - listmonk: [v2.0.0]
 - OS: [Ubuntu]

I can't upload files in media, I received the error (media.go:67: error uploading file: open /uploads/3840x1080-Wallpaper-073.jpg: no such file or directory) 
**here are my media upload setting :**
![image](https://user-images.githubusercontent.com/95730435/145140793-1e0da390-770a-4a6b-9ecd-135789e7e380.png)

**here are my error notification:**
![image](https://user-images.githubusercontent.com/95730435/145140549-f183b52a-4adc-4dc5-b046-d5d72ff9c6c2.png)

FYI, I used the sample docker.compose.yml configuration and have not yet changed the config in config.toml.

**can anyone help?**
thanks regards

**Comment by knadh** [comment]
https://github.com/knadh/listmonk/issues/586

https://github.com/knadh/listmonk/issues/555

**Comment by haritsandrean** [comment]
thanks, it worked when I changed the dir upload path to "/listmonk/uploads"

**Comment by MaximilianKohler** [comment]
I tried `/listmonk/uploads` and `/home/ubuntu/listmonk/uploads` and still get the error...

![uploads dir](https://user-images.githubusercontent.com/3606996/202835612-fc7d8f24-118e-446c-ba58-cd6c8056a506.jpg)

**Comment by knadh** [comment]
Looks like a permission issue. You have created the `uploads` directory with `sudo`, which isn't right. Run `chown -R ubuntu:ubuntu` in `/home/ubuntu/listmonk` to ensure that all files and directories are owned by the user `ubuntu`.

**Comment by MaximilianKohler** [comment]
TLDR: Per https://askubuntu.com/questions/513523/sudo-doesnt-work-etc-sudoers-is-owned-by-uid-1000-should-be-0 **the correct command** seems to be `chown -R ubuntu:ubuntu .`

Maybe not though. 

`sudo chown -R ubuntu:ubuntu /home/ubuntu/listmonk` then `chown -R ubuntu:ubuntu /home/ubuntu/listmonk/` to check, might be correct. 

---

`chown -R ubuntu:ubuntu` did not work. `chown -R ubuntu:ubuntu /` "worked", but a ton of "operation not permitted" errors. Same for `sudo chown -R ubuntu:ubuntu /`

```
ubuntu@listm:~/listmonk$ sudo !!
sudo sudo chown -R ubuntu:ubuntu /
sudo: /etc/sudo.conf is owned by uid 1000, should be 0
sudo: /usr/bin/sudo must be owned by uid 0 and have the setuid bit set
```

I typed `exit`, and now I can't get back in:

> Putty fatal error
> Network error: software caused connection abort

This says I'm F**ked https://stackoverflow.com/questions/68517729/cant-access-aws-ec2-amazon-linux-instance-over-ssh-or-ftp-network-error-softw, but I rebooted the EC2 instance and recovered SSH access. I can login to listmonk in my browser, but the page is blank. 

Back to SSH, I typed `sudo bash` and got:

> ubuntu@listm:~$ sudo bash
> sudo: /etc/sudo.conf is owned by uid 1000, should be 0
> sudo: /usr/bin/sudo must be owned by uid 0 and have the setuid bit set

Same for `sudo` and `sudo -i`. 

I tried the "Debian-based " **Recovery** script here: https://aws.amazon.com/premiumsupport/knowledge-center/ec2-sudo-commands/ but it seems incomplete. IE: I cd'd to "etc" and none of the permissions were changed. I added the 3 lines from here https://repost.aws/questions/QUVkeS7W0CQKWfdWdadkZEbg/accidentally-change-ownership-of-etc-sudoer-back-to-ec-2-user and that changed permissions, but it was incomplete. I still get:

```
sudo: error in /etc/sudo.conf, line 0 while loading plugin "sudoers_policy"
sudo: /usr/libexec/sudo/sudoers.so must be owned by uid 0
sudo: fatal error, unable to load plugins
```

I added `chown -R root:root /usr/libexec/sudo/sudoers.so` and that seemed to work. I have sudo access back but listmonk doesn't load still.

**Comment by MaximilianKohler** [comment]
elasticIP:9000 is working normally. listmonk.mysite.com is blank. Is this fixable?

And BTW, even after changing the permissions to `ubuntu` the upload error still occurs on elaticIP:9000. 

![dir](https://user-images.githubusercontent.com/3606996/202914205-37cb4f78-e61c-40f8-8075-bb5c952f5738.jpg)
